gobblin-temporal/src/main/java/org/apache/gobblin/temporal/cluster/GobblinTemporalTaskRunner.java

```diff
@@ -224,6 +225,9 @@ public void start()
     // Add a shutdown hook so the task scheduler gets properly shutdown
     addShutdownHook();
 
+    // Update authenticator if set
+    AbstractJobLauncher.setDefaultAuthenticator(ConfigUtils.configToProperties(this.clusterConfig));
```

Comment:
`configToProperties` is somewhat expensive, but since this happens only once at start-up and not in the fast path, that should NOT be a large concern here